Sources/Hub/Hub.swift

```diff
@@ -184,6 +193,23 @@ public class LanguageModelConfigurationFromHub {
         )
         return configs
     }
+    
+    func loadConfig(
```

Comment:
Can we call this method from `loadConfig(modelName:hubApi:)` instead of duplicating the code? `snapshot` will return the local folder, so we can load from there as you do in the tests :)